docs/README.md

```diff
@@ -148,41 +333,53 @@ holding `ctrl`/`cmd`.
 
 ### Ansible Lightspeed with watsonx Code Assistant
 
-AI based Ansible code recommendations
+The extension integrates with Ansible Lightspeed with watsonx Code Assistant to
+provide AI-powered features.
 
-- [Getting started](https://docs.redhat.com/en/documentation/red_hat_ansible_lightspeed_with_ibm_watsonx_code_assistant/2.x_latest/html/red_hat_ansible_lightspeed_with_ibm_watsonx_code_assistant_user_guide/set-up-lightspeed_lightspeed-user-guide#set-up-lightspeed_lightspeed-user-guide)
+## **AI-powered Suggestions**
 
-- [Contact](https://matrix.to/#/%23ansible-lightspeed:ansible.im)
+Lightspeed provides inline code suggestions as you type:
 
-## Requirements
+- Press Ctrl+. to trigger suggestions
+- Press Tab to accept a suggestion
+- Press Escape to hide a suggestion
 
-- [Ansible 2.9+](https://docs.ansible.com/ansible/latest/index.html)
-- [Ansible Lint](https://ansible-lint.readthedocs.io/en/latest/) (required,
-  unless you disable linter support; install without `yamllint`)
+## **Playbook Generation**
 
-For Windows users, this extension works perfectly well with extensions such as
-`Remote - WSL` and `Remote - Containers`.
+You can generate complete playbooks from natural language descriptions:
 
-> If you have any other extension providing language support for Ansible, you
-> might need to uninstall it first.
+- Use the command "Ansible: Create a playbook with Ansible Lightspeed"
+- Describe what you want the playbook to do
+- Review and customize the generated playbook
 
-## Configuration
+## **Configuration**
```

Comment:
This looks like it's missing some sections, and that some settings are under different sections than in the extension. If we break this Configuration section into its own page, we can list out everything without crowding the README itself. I'll make a larger recommendation for this below.